src/types/Firestore.ts

```diff
@@ -31,54 +31,65 @@ interface UserVisit {
 export type ValidFirestoreKeys = keyof FirestoreData | keyof FirestoreOrdered;
```

Comment:
I'm assuming these changes are from merging staging into this PR. Not going to deeply read/compare unless told otherwise.